fontforge/scripting.c

```diff
@@ -2374,8 +2375,6 @@ static void bExport(Context *c) {
 	if ( c->curfv->selected[i] && (gid=c->curfv->map->map[i])!=-1 &&
 		SCWorthOutputting(c->curfv->sf->glyphs[gid]) )
 	    ScriptExport(c->curfv->sf,bdf,format,gid,format_spec,c->curfv->map);
```

Comment:
But it can't? `free(tmp)` is only called before the function returns  (line 2361) or at the end of the function (out of the for loop at line 2378)